clipboard/ios/Plugin/Clipboard.swift

```diff
@@ -0,0 +1,72 @@
+import Foundation
+import Capacitor
+
+@objc public class Clipboard: NSObject {
+    
+    @objc enum ContentType: Int {
+        case string
+        case url
+        case image
+    }
+    
+    @objc func write(_ label: ContentType, _ content: String) {
+        switch label {
+        case ContentType.string:
+                UIPasteboard.general.string = content
+        case ContentType.url:
+            if let url = URL(string: content) {
+                UIPasteboard.general.url = url
+            }
+        case ContentType.image:
+            if let data = Data(base64Encoded: getCleanData(content)) {
+                let image = UIImage(data: data)
+                CAPLog.print("Loaded image", image!.size.width, image!.size.height)
+                UIPasteboard.general.image = image
+            } else {
+                CAPLog.print("Unable to encode image")
+            }
+        default:
+            CAPLog.print("Unknown clipboard content type")
+        }
+    }
+
+    @objc func read() -> [String: Any] {
+        if UIPasteboard.general.hasStrings {
+            return [
+                "value": UIPasteboard.general.string!,
+                "type": "text/plain"
+            ]
+            
+        }
```

Comment:
Rather than checking `hasStrings`, `hasURLs`, etc. before accessing the data, it would be better to try to access the data with an optional binding. In addition to being simpler, it will also eliminate the force unwrap (which is legal since the value can be assumed to exist but causes linting problems).
```suggestion
        if if let stringValue = UIPastboard.general.string {
            return [
                "value": stringValue,
                "type": "text/plain"
            ]
        }
```
And so on for the other types.